src/client/Microsoft.Identity.Client/PlatformsCommon/Shared/SimpleHttpClientFactory.cs

```diff
@@ -12,25 +12,26 @@ namespace Microsoft.Identity.Client.PlatformsCommon.Shared
     /// </summary>
     /// <remarks>
     /// This implementation is not suitable for high-scale applications / confidential client scenarios
-    /// because creating new HttpClients leads to a port exhaustion issue. 
+    /// because a static HttpClient might run into DNS issues (and creating new HttpClients leads to a port exhaustion issue).
     /// Mobile platforms should use HttpClientHandlers that are platform specific.
-    /// See platform specific implementations for details on these issues.
+    /// .NET Core should use the IHttpClientFactory, but MSAL cannot take a dependency on it as it has too many dependencies itself.
     /// </remarks>
     internal class SimpleHttpClientFactory : IMsalHttpClientFactory
     {
-        private static readonly Lazy<HttpClient> _httpClient = new Lazy<HttpClient>(() => InitializeClient());
+        private static readonly Lazy<HttpClient> s_httpClient = new Lazy<HttpClient>(() => InitializeClient());
 
         private static HttpClient InitializeClient()
         {
-            var httpClient = new HttpClient(new HttpClientHandler() { UseDefaultCredentials = true });
+            var httpClient = new HttpClient(new HttpClientHandler() { 
+                /* important for IWA */ UseDefaultCredentials = true });
```

Comment:
This was missing in the NetCore client factory. Unfortunately it cannot be added because of a bug on .net core 2.1 (which was fixed on .net core 3.1 only). So we have to revert to the simple factory. This should be fine because on .NET core, our main customers are using Microsoft.Identity.Web which injects a performant httpclient factory that is part of asp.net core.